packages/node/src/evm/handlers/initialize-provider.test.ts

```diff
@@ -15,6 +15,7 @@ import * as fixtures from '../../../test/fixtures';
 
 describe('initializeProvider', () => {
   it('fetches, maps and authorizes requests', async () => {
+    jest.setTimeout(30_000);
```

Comment:
do we want to leave this?

Reply:
Yes, I had some problems running this locally. It doesn't hurt to add this anyway.